### ISSUE TITLE
Can I prevent yargs .describe feature from showing auto-generated info about type, default, etc?

### ISSUE THREAD

**Issue by BadIdeaException** [yargs/yargs#2153]
When using yargs [.describe](https://yargs.js.org/docs/#api-reference-describekey-desc) method to generate usage information, is there some way to prevent yargs from auto-adding information about type, default value, choices etc.?

I.e. the following
```
        .option('foo', {
            type: 'string',
            choices: ['bar','baz'],
            required: true,
            default: 'bar',
            description: 'The most important option of all'
        })
```
results in this output when doing --help:
```
--foo      The most important option of all      [string] [required] [choices: "bar", "baz"] [default: "bar"]
```
I have to confess I find that to be a bit ugly, and it is out of line with how help texts are generally formatted in linux. Also, on longer description texts that include line breaks, it breaks formatting.

I haven't found anything in the docs. Is there some way to suppress generation of the bracketed `[]` parts, without having to completely give up using the describe feature?

**Comment by i-siso** [comment]
@bcoe Thanks for the awesome awesome lib. As mentioned in the issue, and also the open #2156, any update on having this change introduced?
Apart from the programmers, the end users generally don't care about the type or can get confused by the mention of the type in help!

**Comment by bcoe** [comment]
@i-siso thank you 😄 

Unfortunately no updates, I've had no spare time I can put towards OSS recently. I think it's just a matter of dusting off @jly36963's PR.

If @jly36963 thinks it's ready, I'm happy to see it land.

**Comment by bcoe** [comment]
There is now an option to turn off type information, implemented in #2156 

Additional usage configuration settings can be added as needed. This will be released in the next update to yargs.

**Comment by 06000208** [comment]
Can this feature be released in yargs@next as a bleeding edge feature, per the readme? It [appears](https://www.npmjs.com/package/yargs?activeTab=versions) that yargs@next is several versions behind latest, and does not include it as of writing
